sap-2-tier-marketplace-image/shared/loadbalancer.json

```diff
@@ -60,45 +60,63 @@
             "type": "string",
             "metadata": {
                 "description": "The URL to github where the subtemplates are stored"
-            }
+            },
+            "defaultValue": ""
+        },
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+            },
+            "defaultValue": ""

```

Comment:
will need a defaultValue here once CI is fixed...  To test it now you can use the script in the root of the repo.